File: Apps/Components/CMC/2099 - Interface/Controls/ComboBox/cbo.vb
Namespace UI.Control
    ''' <project>CMCC</project>
    ''' <author>Ardha Gp</author>
    ''' <summary>
    ''' Custom combo box
    ''' </summary>
    <System.ComponentModel.DesignerCategory("Code")>
    Public Class Cbo
        Inherits System.Windows.Forms.ComboBox

        <System.Runtime.Versioning.SupportedOSPlatform("windows")>
        Public Sub New()
            InitializeComponent()
            MyBase.Font = Component.Properties.globalFontLbl
            MyBase.DropDownStyle = System.Windows.Forms.ComboBoxStyle.DropDownList
            MyBase.FlatStyle = System.Windows.Forms.FlatStyle.Flat
            MyBase.DoubleBuffered = True
        End Sub
    End Class
End Namespace

File: Apps/Core/CMCc/2099 - Interface/Controls/Cbo.cs
namespace CMCc.UI.Controls
{
    public partial class CboCs : System.Windows.Forms.ComboBox
    {
        public CboCs()
        {
            InitializeComponent();
            this.Font = CMCc.Component.Properties.Global.globalFontLbl;
            this.DropDownStyle = System.Windows.Forms.ComboBoxStyle.DropDownList;
            this.FlatStyle = System.Windows.Forms.FlatStyle.Flat;
            this.DoubleBuffered = true;
        }

    }
}
